src/test/kotlin/com/atlassian/performance/tools/sshubuntu/api/ssh/SshUbuntuTest.kt

```diff
@@ -73,4 +76,14 @@ class SshUbuntuTest {
 
         assertThat(privileged).isTrue()
     }
+
+    @Test
+    fun shouldAptGetWork() {
```

Comment:
> SSH Ubuntu should apt-get work

Let's use better English here. Maybe "SSH Ubuntu should install via apt-get" or "SSH Ubuntu should install software-properties-common".